pyproject.toml

```diff
@@ -336,6 +338,14 @@ line-length = 100
 target-version = "py310"
 
 [tool.ruff.lint]
+external = [
+  "DOC" # pydoclint
```

Comment:
This config is synced with other repos, it will need to be changed there as well (all but nav, lint, compat) LMK, I cna do it.